.pre-commit-config.yaml

```diff
@@ -12,15 +13,17 @@ repos:
     rev: 5.8.0
     hooks:
       - id: isort
+        args: ["--profile", "black"]
+        exclude: ^blackboxopt/examples/
```

Comment:
Can we then have the same hook again that covers the examples directory? Otherwise, for unsorted imports in the examples, I'd expect to run into issues only when the matching action is run.

Reply:
We have? (L15-17) Or do you mean something differently?